tests/framework/common.go

```diff
@@ -1,4 +1,4 @@
-package e2e
+package e2ef
```

Comment:
Any particular reason this isn't `package framework` to match the directory name?

Reply:
It was initially like that... and changed it. Can't even reason why.
I'll fix.